yoyodyne/models/lstm.py

```diff
@@ -243,31 +229,37 @@ def beam_decode(
             # Constrains the next step to beamsize.
             for (
                 predictions,
```

Comment:
This was a poor choice by me. Can we maintain the name `logits` instead of changing it to `predictions` here? Sorry for the drive-by.